Create mobile menu functionality

aceDavon opened the pull request:
Add javascript event listener for click to display the mobile menu and
hide on click toggle

aceDavon committed:
Create mobile menu functionality

Add javascript event listener for click to display the mobile menu and
hide on click toggle

SurafelGetachewT committed:
Link the menu items with the page sections

aceDavon committed:
Make styles for the mobile menu

Make styles for the mobile menu try to match the figma template design,
Apply transition effect on dropdown of the mobile menu

aceDavon committed:
Fix linter errors on style.css

youmari commented on index.html:
```diff
@@ -14,32 +14,60 @@
         </li>
       </ul>
+      <div class="menu-container" id="topMenu">
+        <div class="close" id="close">
+          <img src="./img/icons/close_white_48dp.svg" alt="close-icon" />
+        </div>
```
- [ ] please make sure that the close button has the same size as the one in the Figma design and also try to align it in the right place as in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/179769767-4486bd0c-406d-4801-ae67-bc9e528a3928.png)

youmari commented on index.html:
```diff
@@ -14,32 +14,60 @@
+        </div>
+        <ul class="drop-menu">
+          <li>
+            <a href="#portfolio">Portfolio</a>
+            <img
+              src="./img/icons/keyboard_arrow_right_white_48dp.svg"
+              alt="pointer1"
+            />
+          </li>
+          <li>
+            <a href="#about">About</a>
+            <img
+              src="./img/icons/keyboard_arrow_right_white_48dp.svg"
+              alt="pointer2"
+            />
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+            <img
+              src="./img/icons/keyboard_arrow_right_white_48dp.svg"
+              alt="pointer3"
+            />
+          </li>
```
- [ ] please try to use the same font-size, font family, space between the links in order to match the Figma design 
![image](https://user-images.githubusercontent.com/61790587/179770066-e2353493-0dfd-47e8-8fd1-aff38e0557ef.png)

youmari reviewed (changes requested):
Hi @aceDavon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great title and short summary 
✔️ No linter error
✔️ Github flow 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

aceDavon committed:
Fix issues raised from reviewer

Import google fonts to use the roboto font on the style.css file, adjust
the width of icons and positioning.

Lucas-Masaba reviewed (approved):
Hello @aceDavon and @SurafelGetachewT  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![almost-there](https://media.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif)

## Highlights 🍾 
- Linter checks passing ✔️ 
- Made changes according to previous reviewer 👍 
## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
